<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MapTab</class>
 <widget class="QWidget" name="MapTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>503</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QTabWidget" name="widgetMap">
     <property name="styleSheet">
      <string notr="true">QTabWidget::tab-bar { alignment: center; } </string>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabThermal">
      <attribute name="title">
       <string>Thermal Diagram</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_9">
       <item row="5" column="0" colspan="4">
        <layout class="QHBoxLayout" name="layoutThermalTimeLine">
         <item>
          <widget class="QLabel" name="labelThermalTimeLine">
           <property name="text">
            <string>TimeLine</string>
           </property>
           <property name="buddy">
            <cstring>sliderThermalTimeLine</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="sliderThermalTimeLine">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="labelThermalTime">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="0" colspan="4">
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0">
          <widget class="QLabel" name="labelThermalTimeFrom">
           <property name="text">
            <string>From</string>
           </property>
           <property name="buddy">
            <cstring>editThermalTimeFrom</cstring>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QDateTimeEdit" name="editThermalTimeTo">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="dateTime">
            <datetime>
             <hour>23</hour>
             <minute>59</minute>
             <second>0</second>
             <year>2016</year>
             <month>11</month>
             <day>1</day>
            </datetime>
           </property>
          </widget>
         </item>
         <item row="0" column="5">
          <widget class="QSpinBox" name="spinThermalStepTime">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>100</number>
           </property>
           <property name="value">
            <number>30</number>
           </property>
          </widget>
         </item>
         <item row="1" column="4">
          <widget class="QLabel" name="labelThermalStep">
           <property name="text">
            <string>spaceStep</string>
           </property>
          </widget>
         </item>
         <item row="1" column="5">
          <widget class="QSpinBox" name="spinThermalStep">
           <property name="value">
            <number>30</number>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QLabel" name="labelThermalTimeTo">
           <property name="text">
            <string>To</string>
           </property>
           <property name="buddy">
            <cstring>editThermalTimeTo</cstring>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QDateTimeEdit" name="editThermalTimeFrom">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="dateTime">
            <datetime>
             <hour>0</hour>
             <minute>0</minute>
             <second>0</second>
             <year>2016</year>
             <month>11</month>
             <day>1</day>
            </datetime>
           </property>
          </widget>
         </item>
         <item row="0" column="4">
          <widget class="QLabel" name="labelThermalStepTime">
           <property name="text">
            <string>timeStep</string>
           </property>
           <property name="buddy">
            <cstring>spinThermalStepTime</cstring>
           </property>
          </widget>
         </item>
         <item row="1" column="0" colspan="2">
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QRadioButton" name="radioThermalOverall">
             <property name="text">
              <string>overall</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="radioThermalMoment">
             <property name="text">
              <string>moment</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
             <property name="autoRepeat">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="1" column="2" colspan="2">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Red: entry  Blue: exit</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QQuickWidget" name="plotThermalOrigin">
           <property name="resizeMode">
            <enum>QQuickWidget::SizeRootObjectToView</enum>
           </property>
           <property name="source">
            <url>
             <string>qrc:/Maps/map.qml</string>
            </url>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="1">
        <widget class="Line" name="lineThermal">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QQuickWidget" name="plotThernalDestination">
           <property name="resizeMode">
            <enum>QQuickWidget::SizeRootObjectToView</enum>
           </property>
           <property name="source">
            <url>
             <string>qrc:/Maps/map.qml</string>
            </url>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabTaxiFlow">
      <attribute name="title">
       <string>Taxi Flow</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_13">
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="labelFlowTimeFrom">
           <property name="text">
            <string>From</string>
           </property>
           <property name="buddy">
            <cstring>editFlowTimeFrom</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDateTimeEdit" name="editFlowTimeFrom">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="dateTime">
            <datetime>
             <hour>0</hour>
             <minute>0</minute>
             <second>0</second>
             <year>2016</year>
             <month>11</month>
             <day>1</day>
            </datetime>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="labelFlowTimeTo">
           <property name="text">
            <string>To</string>
           </property>
           <property name="buddy">
            <cstring>editFlowTimeTo</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDateTimeEdit" name="editFlowTimeTo">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="dateTime">
            <datetime>
             <hour>23</hour>
             <minute>59</minute>
             <second>0</second>
             <year>2016</year>
             <month>11</month>
             <day>1</day>
            </datetime>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="labelFlowStep">
           <property name="text">
            <string>Step</string>
           </property>
           <property name="buddy">
            <cstring>spinFlowStep</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spinFlowStep">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>100</number>
           </property>
           <property name="value">
            <number>50</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <widget class="QQuickWidget" name="plotFlow">
         <property name="resizeMode">
          <enum>QQuickWidget::SizeRootObjectToView</enum>
         </property>
         <property name="source">
          <url>
           <string>qrc:/Maps/map.qml</string>
          </url>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="labelFlowTimeLine">
           <property name="text">
            <string>TimeLine</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="sliderFlowTimeLine">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="labelFlowTime">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabRoute">
      <attribute name="title">
       <string>Route Planning</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_16">
       <item row="0" column="2">
        <layout class="QGridLayout" name="layoutRoute">
         <item row="2" column="0" colspan="2">
          <widget class="QPushButton" name="buttonRoutePlanning">
           <property name="text">
            <string>Route Planning</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="labelRoute">
           <property name="text">
            <string>Route</string>
           </property>
           <property name="buddy">
            <cstring>listRoute</cstring>
           </property>
          </widget>
         </item>
         <item row="5" column="0" colspan="2">
          <widget class="QListWidget" name="listRoute">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item row="1" column="0" colspan="2">
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="1" column="1">
            <widget class="QLineEdit" name="editRouteDestination"/>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="labelRouteDestination">
             <property name="text">
              <string>Destination</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="labelRouteOrigin">
             <property name="text">
              <string>Origin</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="editRouteOrigin"/>
           </item>
           <item row="1" column="2">
            <widget class="QPushButton" name="buttonRouteDestination">
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="../resources.qrc">
               <normaloff>:/Maps/marker.png</normaloff>:/Maps/marker.png</iconset>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QPushButton" name="buttonRouteOrigin">
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="../resources.qrc">
               <normaloff>:/Maps/marker.png</normaloff>:/Maps/marker.png</iconset>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item row="0" column="0">
        <widget class="QQuickWidget" name="plotRoute">
         <property name="resizeMode">
          <enum>QQuickWidget::SizeRootObjectToView</enum>
         </property>
         <property name="source">
          <url>
           <string>qrc:/Maps/map.qml</string>
          </url>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="Line" name="lineRoute">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QQuickWidget</class>
   <extends>QWidget</extends>
   <header location="global">QtQuickWidgets/QQuickWidget</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>widgetMap</sender>
   <signal>currentChanged(int)</signal>
   <receiver>MapTab</receiver>
   <slot>loadMap()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>111</x>
     <y>24</y>
    </hint>
    <hint type="destinationlabel">
     <x>38</x>
     <y>4</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinThermalStep</sender>
   <signal>editingFinished()</signal>
   <receiver>MapTab</receiver>
   <slot>loadMap()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>439</x>
     <y>89</y>
    </hint>
    <hint type="destinationlabel">
     <x>349</x>
     <y>0</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinFlowStep</sender>
   <signal>editingFinished()</signal>
   <receiver>MapTab</receiver>
   <slot>setFlowRange()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>481</x>
     <y>88</y>
    </hint>
    <hint type="destinationlabel">
     <x>441</x>
     <y>0</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>editThermalTimeFrom</sender>
   <signal>editingFinished()</signal>
   <receiver>MapTab</receiver>
   <slot>setThermalRange()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>65</x>
     <y>53</y>
    </hint>
    <hint type="destinationlabel">
     <x>7</x>
     <y>65</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>editThermalTimeTo</sender>
   <signal>editingFinished()</signal>
   <receiver>MapTab</receiver>
   <slot>setThermalRange()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>227</x>
     <y>53</y>
    </hint>
    <hint type="destinationlabel">
     <x>464</x>
     <y>8</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinThermalStepTime</sender>
   <signal>editingFinished()</signal>
   <receiver>MapTab</receiver>
   <slot>setThermalRange()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>439</x>
     <y>53</y>
    </hint>
    <hint type="destinationlabel">
     <x>500</x>
     <y>27</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sliderThermalTimeLine</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MapTab</receiver>
   <slot>selectedThermalTime()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>95</x>
     <y>235</y>
    </hint>
    <hint type="destinationlabel">
     <x>5</x>
     <y>207</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sliderThermalTimeLine</sender>
   <signal>sliderReleased()</signal>
   <receiver>MapTab</receiver>
   <slot>loadMap()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>95</x>
     <y>235</y>
    </hint>
    <hint type="destinationlabel">
     <x>5</x>
     <y>249</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>editFlowTimeFrom</sender>
   <signal>editingFinished()</signal>
   <receiver>MapTab</receiver>
   <slot>setFlowRange()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>83</x>
     <y>72</y>
    </hint>
    <hint type="destinationlabel">
     <x>4</x>
     <y>55</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>editFlowTimeTo</sender>
   <signal>editingFinished()</signal>
   <receiver>MapTab</receiver>
   <slot>setFlowRange()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>246</x>
     <y>71</y>
    </hint>
    <hint type="destinationlabel">
     <x>5</x>
     <y>31</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sliderFlowTimeLine</sender>
   <signal>sliderReleased()</signal>
   <receiver>MapTab</receiver>
   <slot>loadMap()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>118</x>
     <y>249</y>
    </hint>
    <hint type="destinationlabel">
     <x>6</x>
     <y>238</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sliderFlowTimeLine</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MapTab</receiver>
   <slot>selectedFlowTime()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>149</x>
     <y>246</y>
    </hint>
    <hint type="destinationlabel">
     <x>7</x>
     <y>192</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioThermalOverall</sender>
   <signal>clicked()</signal>
   <receiver>MapTab</receiver>
   <slot>loadMap()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>73</x>
     <y>94</y>
    </hint>
    <hint type="destinationlabel">
     <x>4</x>
     <y>141</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioThermalMoment</sender>
   <signal>clicked()</signal>
   <receiver>MapTab</receiver>
   <slot>loadMap()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>258</x>
     <y>95</y>
    </hint>
    <hint type="destinationlabel">
     <x>503</x>
     <y>63</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>loadMap()</slot>
  <slot>setThermalRange()</slot>
  <slot>selectedThermalTime()</slot>
  <slot>setFlowRange()</slot>
  <slot>selectedFlowTime()</slot>
 </slots>
</ui>
